src/objects.js

```diff
@@ -1,41 +1,52 @@
 const createPerson = (name, age) => {
-  // your code here
+  this.name = name;
+  this.age = age;
+  return { name, age };
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  return object.hasOwnProperty(property);
 };
 
 const isOver65 = person => {
-  // your code here
+  return person.age > 65;
 };
 
 const getAges = people => {
-  // your code here
+  const ageArray = people.map(person => person.age);
+  return ageArray;
 };
 
 const findByName = (name, people) => {
-  // your code here
+  return people.find(person => person.name === name);
 };
 
 const findHondas = cars => {
-  // your code here
+  return cars.filter(car => car.manufacturer === 'Honda');
 };
 
 const averageAge = people => {
-  // your code here
+  const ageArr = people.map(person => person.age)
+  const averageAge = ageArr.reduce((a, b) => (a + b));
+  return averageAge / people.length;
 };
 
 const createTalkingPerson = (name, age) => {
-  // your code here
+  return {
+    name: name,
+    age: age,
```

Comment:
You can use the object shorthand here, does exactly the same thing as you have here just makes it a bit more concise:
```
    name,
    age,
```